cmd/juju/crossmodel/offer.go

```diff
@@ -102,7 +106,17 @@ func (c *offerCommand) Run(_ *cmd.Context) error {
 	if err != nil {
 		return err
 	}
-	return params.ErrorResults{results}.Combine()
+	err = params.ErrorResults{results}.Combine()
+	if err == nil {
```

Comment:
this feels too clever, can you please just do

```go
if err := params.ErrorResults{results}.Combine(); err != nil {
    return err
}
```

that way you're keepin the return err close to where it was checked, and it doesn't risk getting messed up